pycaret/internal/pipeline.py

```diff
@@ -79,6 +80,42 @@ def __init__(self, steps, *, memory=None, verbose=False):
         self._fit_vars = set()
```

Comment:
Just want to double check that `class Pipeline(imblearn.pipeline.Pipeline)` is the class that's used for all PyCaret classification and regression models created by `compare_models()` or `create_model()`?

Reply:
That is my understanding.
Each `setup()` functions initialize ``pycaret.internal.pipeline.Pipeline` in following rows.
- regression
    - regression/oop.py: 678
- classification
    - classification/oop.py: 709
- anomaly/clustering
    - unsupervised_experiment.py: 694